decentraland/unity-renderer
# Feat: add unsupported bodyshape toast for wearables

## What does this PR change?

Fix #2973 
Adds a small toast to give feedback on unusable wearables due to bodyshape

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/wearable-feedback
2. Open the backpack
3. Based on the body shape verify which wearables are grayed out and which not
4. Verify that hovering with the mouse shows the correct toast

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
